src/main/java/uk/gov/hmcts/ccd/data/casedetails/search/SearchQueryFactoryOperation.java

```diff
@@ -28,25 +28,26 @@
     @PersistenceContext
     private final EntityManager entityManager;
```

Comment:
`CaseDetailsEndpoint` is untouched. This means that we support ordering by custom case fields only on searches driven by the UI and coming from the def file?
Couldn't we also change `searchCases` in `CaseDetailsEndpoint` so that we can use this new feature also on the API?

Reply:
hmm.. i don't know it's required there. Probably question to Steve ? 
P.S : originally this feature requested for workbasket results and then we applied to both workbasket / search tabs config. So, for searchCases in CaseDetailsEndpoint which config is applied?

Reply:
I was not suggesting to have CaseDetailsEndpoint retrieve the sort options from the definition. What I was thinking was to extend the search api (exposed by CaseDetailsEndpoint) to be able to specify sorting on metadata and case fields as query parameters. If we do this, then this feature can be used also by some Service that today calls the search API programmatically. It won't cost much additional work. 
To me it feels very weird via UI we can sort searches but on the API we can't... 
I'll raise the point with Steve

Reply:
Spoke to Steve. We can implement this in the future if a Service requires to do sorting via API